webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/resource/style-ui.scss

```diff
@@ -554,5 +554,22 @@ $background_color_14: #dddddd;
       margin-top: 25px;
     }
   }
+  #span_accept_btn {
```

Comment:
Better use class selectors instead of ID selectors. See comment above on avoiding fixed IDs for elements.